frontend/src/scenes/session-recordings/SessionRecordingsPlaylist.tsx

```diff
@@ -1,123 +1,197 @@
 import { LemonButton } from '@posthog/lemon-ui'
-import { IconChevronLeft, IconChevronRight } from 'lib/components/icons'
+import { IconChevronLeft, IconChevronRight, IconSchedule } from 'lib/components/icons'
 import { SessionRecordingsFilters } from './filters/SessionRecordingsFilters'
+import clsx from 'clsx'
+import { Tooltip } from 'lib/components/Tooltip'
+import { LemonSkeleton } from 'lib/components/LemonSkeleton'
+import { LemonTableLoader } from 'lib/components/LemonTable/LemonTableLoader'
 
 interface SessionRecordingsTableProps {
     personUUID?: string
     isPersonPage?: boolean
 }
 
+const DurationDisplay = ({ duration }: { duration: number }): JSX.Element => {
+    const formattedDuration = colonDelimitedDuration(duration)
+    const parts = formattedDuration.split(':')
+
+    return (
+        <span className="flex items-center gap-1">
+            <IconSchedule className="text-lg" />
+            <span>
+                <span className={parts[0] === '00' ? 'opacity-50' : ''}>{parts[0]}:</span>
+                <span className={parts[0] === '00' && parts[1] === '00' ? 'opacity-50' : ''}>{parts[1]}:</span>
+                {parts[2]}
+            </span>
+        </span>
+    )
+}
+
 export function SessionRecordingsPlaylist({ personUUID }: SessionRecordingsTableProps): JSX.Element {
-    const sessionRecordingsListLogicInstance = sessionRecordingsListLogic({ personUUID, isPlaylist: true })
+    const logic = sessionRecordingsListLogic({ personUUID })
     const { sessionRecordings, sessionRecordingsResponseLoading, hasNext, hasPrev, activeSessionRecording, offset } =
-        useValues(sessionRecordingsListLogicInstance)
-    const { openSessionPlayer, loadNext, loadPrev } = useActions(sessionRecordingsListLogicInstance)
+        useValues(logic)
+    const { setSelectedRecordingId, loadNext, loadPrev } = useActions(logic)
     const playlistRef = useRef<HTMLDivElement>(null)
 
-    const columns: LemonTableColumns<SessionRecordingType> = [
-        {
-            title: 'Recordings',
-            render: function RenderPlayButton(_: any, sessionRecording: SessionRecordingType) {
-                return (
-                    <div>
-                        {asDisplay(sessionRecording.person, 25)}
-                        <div>
-                            <span>
-                                <TZLabel
-                                    time={sessionRecording.start_time}
-                                    formatDate="MMMM DD, YYYY"
-                                    formatTime="h:mm A"
-                                />
-                                {` · `}
-                                {colonDelimitedDuration(sessionRecording.recording_duration)}
-                            </span>
-                        </div>
-                    </div>
-                )
-            },
-        },
-    ]
+    const onRecordingClick = (recording: SessionRecordingType): void => {
```

Comment:
should we disabled scrollTo behavior on non-sm screen sizes?

This currently makes for some unnecessary scrolling:


https://user-images.githubusercontent.com/13460330/195633841-5966fd2d-4357-41f6-a2d8-274146beb5f0.mov

Reply:
Video won't load for some reason. Can you try again / post it to slack?

Reply:
Ah now it works. Yeah I think we should only scroll if the player is higher up.

Reply:
Updated it to only scroll if the window is scrolled below the top point